test/Tester/ActivationServices/IStorageActivationService.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+using Orleans.Runtime;
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Orleans;
+
+namespace Tester
+{
+    public interface IStorageActivationService<TState>
```

Comment:
I created this for the prototype to have a more concrete picture of what it would look like in prod. Do you think we should add it with this name or have Test somewhere inside the name?